### PR TITLE
grand Go SDK update

### PR DESCRIPTION
**Pull request by dicej**
- Add `outbound-redis` set operations and `execute` function
- Add `key-value` support
- Remove `*_free` calls since they were buggy and unnecessary given that we tell `tinygo` to leak anyway
- Remove `-wasm-abi=generic` option since it's been removed as of `tinygo` 0.27

### CODE REVIEW

File: examples/tinygo-outbound-redis/main.go [hunk 1]
```diff
@@ -62,6 +66,80 @@ func init() {
 			w.Write([]byte(strconv.FormatInt(payload, 10)))
 			w.Write([]byte("\n"))
 		}
+
+		if _, err := redis.Sadd(addr, "myset", []string{"foo", "bar"}); err != nil {
```
[the remark this answers is not in the document]
Comment (dicej) [reply examples/tinygo-outbound-redis/main.go:70]: I believe that's the second time you've made that joke.

[the remark this answers is not in the document]
Comment (lann) [reply examples/tinygo-outbound-redis/main.go:70]: WELL ITS STILL JUST AS FUNNY

**Comment by dicej** [comment]
Somehow I forget to sign commits 50% of the time.  I blame `git rebase`.

**Comment by lann** [comment]
> Somehow I forget to sign commits 50% of the time. I blame `git rebase`.

This might appeal to you (all my Fermyon repos are under `~/f/`):

`~/.gitconfig`
```
[includeIf "gitdir:~/f/"]
  path = ~/f/gitconfig.inc
```
`~/f/gitconfig.inc`
```
[user]
email = lann.martin@fermyon.com
signingkey = C90E9024FB60F7FC74F39250A30EEE9905396642

[commit]
gpgsign = true
signOff = true
```

**Review by lann — APPROVED** [review]

**Review by adamreese — APPROVED** [review]
Awesome job @dicej

File: sdk/go/key-value/key-value.go [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+	ErrorKindNoSuchStore
+	ErrorKindAccessDenied
+	ErrorKindInvalidStore
+	ErrorKindNoSuchKey
+	ErrorKindIo
+)
+
+type Error struct {
+	Kind ErrorKind
+	Val any
```
Comment (adamreese) [inline sdk/go/key-value/key-value.go:26]: you might need to change this to `interface{}`

Comment (adamreese) [reply sdk/go/key-value/key-value.go:26]: As well as the rest of type `any`

Comment (adamreese) [reply sdk/go/key-value/key-value.go:26]: Or you'll need to bump the go (not tinygo) version

File: examples/tinygo-outbound-redis/main.go [hunk 3]
```diff
@@ -4,6 +4,8 @@ import (
 	"net/http"
 	"os"
 	"strconv"
+	"reflect"
```
[the remark this answers is not in the document]
Comment (dicej) [reply examples/tinygo-outbound-redis/main.go:7]: Good catch.  I had hoped Redis was using a sorted set internally and thus could expect a consistent order, but clearly that's not true.  I'll make the test more robust.

**Pull request merged**